apps/web/hooks/useLinkResolver/useLinkResolver.ts

```diff
@@ -194,6 +194,18 @@ export const routesTemplate = {
     is: '/[slug]',
     en: '/en/[slug]',
   },
+  universitysearchdetails: {
+    is: '/haskolanam/[id]',
+    en: '/en/haskolanam/[id]',
+  },
+  universitysearchcomparison: {
+    is: '/haskolanam/samanburdur',
+    en: '/en/haskolanam/comparison',
+  },
+  universitysearch: {
+    is: '/haskolanam',
```

Comment:
Just to let you know we've got a 404 redirect in the CMS which is pointing /haskolanam so when this goes live it'll appear instead

Reply:
Yes we will change that when this goes out